doc/en/components/charts/features/chart-annotations.md

```diff
@@ -20,17 +22,33 @@ This topic provides conceptual information about the annotations and overlay int
 
 <div class="divider--half"></div>
 
+Like this sample? Get access to our complete $PlatformShort$ toolkit and start building your own apps in minutes. <a href="{environment:infragisticsBaseUrl}/products/$ProductSpinal$/download">Download it for free.</a>
+
 ## $PlatformShort$ Crosshair Layer
 
-The Crosshair Layer of the category chart provides crossing lines that meet at the actual value of every targeted series. Crosshair types include: Horizontal, Vertical, and Both.  The Crosshairs can also be configured to snap to data points by setting the `CrosshairsSnapToData` property to true, otherwise the crosshairs will be interpolated between data points. Annotations can also be enabled to display the crosshair's value along the axis.
+The Crosshair Layer renders as crossing lines intersecting at the actual value of every series that they are configured to target with each series rendering a separate set of lines. 
 
-```razor
-<CategoryChart Height="100%" Width="100%"
-    DataSource="@DataSource"
-    CrosshairsDisplayMode="CrosshairsDisplayMode.Horizontal"
-    CrosshairsAnnotationEnabled="true"
-    CrosshairsSnapToData="true"/>
-```
+Crosshair types include:
+- Horizontal
+- Vertical
+- Both
+
+The chart's crosshairs can also be configured to snap to data points by setting the `CrosshairsSnapToData` property to true, otherwise the crosshairs will be interpolated between data points. Annotations can also be enabled to display the crosshair's value along the axis. 
+
+You can configure the crosshair layer so that the layer will only display on one specific series, as by default they will target all series on the chart component. To achieve this, set the `TargetSeries` property.  
+
```

Comment:
in mentionedTypes metadata, you must add a name of series/layer that you mentioning API properties in topics otherwise API links will be broken